extension/solarwindsapmsettingsextension/extension.go

```diff
@@ -33,29 +43,32 @@ func newSolarwindsApmSettingsExtension(extensionCfg *Config, logger *zap.Logger)
 }
 
 func (extension *solarwindsapmSettingsExtension) Start(_ context.Context, _ component.Host) error {
-	extension.logger.Info("Starting up solarwinds apm settings extension")
+	extension.logger.Info("starting up solarwinds apm settings extension")
 	ctx := context.Background()
 	ctx, extension.cancel = context.WithCancel(ctx)
-	var err error
-	extension.conn, err = grpc.NewClient(extension.config.Endpoint, grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{})))
+	systemCertPool, err := x509.SystemCertPool()
```

Comment:
I added code to get the system certificates pool for TLS configuration.

Reply:
I suggest reusing the configuration and helpers that are provided in https://github.com/open-telemetry/opentelemetry-collector/tree/main/config/configgrpc , saves a decent amount of effort and reduces the risk of doing it wrong.

Reply:
Done. It is a good idea. Thanks!